src/clr-angular/forms/checkbox/checkbox.ts

```diff
@@ -19,8 +26,25 @@ export class ClrCheckbox extends WrappedFormControl<ClrCheckboxWrapper> {
     @Optional()
     control: NgControl,
     renderer: Renderer2,
-    el: ElementRef
+    el: ElementRef,
+    @Attribute('clrToggle') private toggle: string
   ) {
     super(vcr, ClrCheckboxWrapper, injector, control, renderer, el);
   }
+
+  ngOnInit() {
+    super.ngOnInit();
+
+    // This works fine normally, but with testing it throws unless its wrapped manually or has try/catch
+    // It basically doesn't seem to wire up the host wrapper container in time for this specific call, but the wrapper is there
+    try {
```

Comment:
The fact that you need an empty try/catch is pretty scary. I'm okay(-ish) shipping this blindly if it works, but it might be hiding a real issue somewhere that I can't see right now.

Reply:
As we discussed, I moved this up into the wrapped control directive with a comment about it.